portality/view/api_v1.py

```diff
@@ -40,8 +34,74 @@ def list_operations():
 def docs():
     return render_template('doaj/api_docs.html')
 
+@blueprint.route("/search/applications/<path:search_query>")
+@api_key_required
+def search_applications(search_query):
+    """
+    Search your applications
+    ---
+    tags:
+      - search
+    description: "Search your applications to the DOAJ"
+    parameters:
+      -
+        name: "search_query"
+        in: "path"
+        required: true
+        type: "string"
+      -
+        name: "page"
+        in: "body":
+        required: false
+        type: "int"
+      -
+        name: "pageSize"
+        in: "body"
+        required: false,
+        type: "int"
+      -
+        name: "sort"
+        in: "body"
+        required: false
+        type: "string"
+      -
+        name: "api_key"
```

Comment:
Ah, I think this is usually used as HTTP basic auth instead of passing the key into the params, so you don't get your API key plastered over your requests, but just get it put in a HTTP header. My fault for not spotting on @Steven-Eardley's PR, I'll discuss in that issue. It's not an issue with *this* PR.